src/bci_build/package/appcontainers.py

```diff
@@ -163,6 +178,8 @@
             entrypoint=["docker-entrypoint.sh"],
             extra_files={
                 "docker-entrypoint.sh": _MARIADB_ENTRYPOINT,
+                "healthcheck.sh": _MARIADB_HEALTHCHECK,
+                "gosu": _MARIADB_GOSU,
```

Comment:
This is _not_ `gosu`, nor is it even properly "`gosu` compatible" -- please give the binary in the image a different name.

(If full `gosu` compatibility is desired, please see https://github.com/tianon/gosu/pull/143 which has been successfully tested against `gosu`'s test suite for all the parsing edge cases I'm aware of and maintain a test suite for.)

Reply:
Thank you! ❤️